account_invoice_refund_option/view/view_account_invoice_refund.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_account_invoice_refund_form_inherit"
+        model="ir.ui.view">
+        <field name="model">account.invoice.refund</field>
+        <field name="inherit_id"
+            ref="account.view_account_invoice_refund" />
+        <field name="arch" type="xml">
+            <xpath expr="//group/group[2]" position="inside">
+                <group>
+                    <div
+                        attrs="{'invisible': [('filter_refund', '!=', 'refund_select_product')]}"
+                        class="oe_grey" colspan="4">
+                        Use this option if you
+                        want to create a draft refund for some products
+                        only.
+                    </div>
+                </group>
+            </xpath>
+            <xpath expr="//group" position="after">
+                <group string="Invoice Lines"
+                    attrs="{'invisible': [('filter_refund', '!=', 'refund_select_product')]}">
+                    <field name="invoice_line_ids" nolabel="1"
+                        options="{'no_create': True}">
+                        <tree string="Bill lines" editable="bottom">
+                            <field name="sequence" widget="handle" />
+                            <field name="product_id" />
+                            <field name="name" />
+                            <field name="account_id" />
+                            <field name="quantity" />
+                            <field name="uom_id"
+                                groups="uom.group_uom" />
+                            <field name="price_unit" />
+                            <field name="discount"
+                                groups="sale.group_discount_per_so_line" />
+                            <field name="invoice_line_tax_ids"
+                                widget="many2many_tags" />
+                            <field name="price_subtotal" />
+                        </tree>
+                    </field>
+                </group>
+            </xpath>
+        </field>
+    </record>
```

Comment:
FYI, there's no need to limit characters on xml lines.
No need to change anythnig here, but it seems you're trying to fit xml files in 80 character lines, and it's not necessary